lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -585,6 +585,16 @@ def test_date_with_self_paced_with_enrollment_before_course_start(self):
         enrollment = CourseEnrollmentFactory(course_id=course.id, mode=CourseMode.AUDIT)
         block = VerifiedUpgradeDeadlineDate(course, enrollment.user)
         self.assertEqual(block.date, expected)
+        self._check_text(block)
+
+    def _check_text(self, upgrade_date_summary):
+        self.assertEqual(upgrade_date_summary.title, 'Upgrade to Verified Certificate')
+        self.assertEqual(
+            upgrade_date_summary.description,
+            'Don\'t miss the opportunity to highlight your new knowledge and skills by earning a verified'
+            ' certificate.'
```

Comment:
Nit: Use `"` on the outside so you don't have to escape the `'` (and also because usually `"` is for human consumption and `'` is for program internals).